waagent

```diff
@@ -5876,7 +5880,7 @@ def main():
     if len(sys.argv) == 1:
         sys.exit(Usage())
 
-    LoggerInit('/var/log/waagent.log','/dev/console')
+    LoggerInit('/dev/stdout',None)
```

Comment:
Hi, @imikushin. What's the purpose of the change of this line? This will break our test.